wled00/wled03_set.ino

```diff
@@ -120,7 +120,7 @@ void handleSettingsSet(AsyncWebServerRequest *request, byte subPage)
   if (subPage == 4)
   {
     buttonEnabled = request->hasArg("BT");
-    irEnabled = request->hasArg("IR");
+    irEnabled = request->arg("IR").toInt();
```

Comment:
this changes the "ON/OFF" checkbox to a value entry field (0 to 4) with 0=disabled, 1=1st remote, 2=2nd remote etc. /// 4 different IR remotes in total.